### PR TITLE
[DOCS] Hopsworks.ai API Keys

### PR DESCRIPTION
**Pull request by maismail**

### CODE REVIEW

File: docs/hopsworksai/api_key.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Hopsworks.ai API Key
+
+Hopsworks.ai allows users to generate an API Key that can be used to authenticate and access the Hopsworks.ai REST APIs.
+
+## Generate an API Key
+
+This section describes how to generate an API Key for Hopsworks.ai. First, login to your Hopsworks.ai account, then click on the Settings tab as shown below:
```
Comment (berthoug) [inline docs/hopsworksai/api_key.md:7]: I would put the first sentence on its own line.
```suggestion
This section describes how to generate an API Key for Hopsworks.ai. 

First, login to your Hopsworks.ai account, then click on the Settings tab as shown below:
```

File: docs/hopsworksai/api_key.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+<p align="center">
+  <figure>
+    <a  href="../../assets/images/hopsworksai/apiKey/api-key-1.png">
+      <img style="border: 1px solid #000" src="../../assets/images/hopsworksai/apiKey/api-key-1.png" alt="Click on Settings">
+    </a>
+    <figcaption>Click on the Settings tab</figcaption>
+  </figure>
+</p>
+
+Click on the API Key tab, and then click on *Generate API Key* button:
```
Comment (berthoug) [inline docs/hopsworksai/api_key.md:18]:
```suggestion
Click on the API Key tab, and then click on the *Generate API Key* button:
```

File: docs/hopsworksai/api_key.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+<p align="center">
+  <figure>
+    <a  href="../../assets/images/hopsworksai/apiKey/api-key-2.png">
+      <img style="border: 1px solid #000" src="../../assets/images/hopsworksai/apiKey/api-key-2.png" alt="Generate API Key">
+    </a>
+    <figcaption>Generate an API Key</figcaption>
+  </figure>
+</p>
+
+Copy the generated API Key and store it in a secure location.
```
Comment (berthoug) [inline docs/hopsworksai/api_key.md:29]: I would add a warning there to say that it won't be shown again and that if you lose it you will have to delete it and create a new one.

File: docs/hopsworksai/api_key.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+## Use the API Key
+
+To access the Hopsworks.ai REST APIs, you should pass the API key as a header **x-api-key** when executing requests on Hopsworks.ai as shown below:
+
+```bash
+curl -XGET -H "x-api-key: <YOUR API KEY>" https://api.hopsworks.ai/api/clusters
+```
+
+Alternatively, you can use your API Key with the [Hopsworks.ai terraform provider](https://registry.terraform.io/providers/logicalclocks/hopsworksai/latest) to manage your Hopsworks clusters using [terraform](https://www.terraform.io/).
```
Comment (berthoug) [inline docs/hopsworksai/api_key.md:48]: The link to the provider is broken. I guess it is normal. Is it?

File: docs/hopsworksai/api_key.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## Delete your API Key
+
+This section describes how to delete your Hopsworks.ai API Key. First, login to your Hopsworks.ai account, click on the Settings tab, then click on the API Key tab, and finally click on *Delete API Key* as shown below:
```
Comment (berthoug) [inline docs/hopsworksai/api_key.md:52]: Given the shortness of the section and the title I don't think that it is needed to describe what the section is for.
```suggestion
First, login to your Hopsworks.ai account, click on the Settings tab, then click on the API Key tab, and finally click on *Delete API Key* as shown below:
```

File: docs/hopsworksai/api_key.md (hunk 1 above)
Comment (berthoug) [reply docs/hopsworksai/api_key.md:7]: Given the shortness of the section and the title I think you could also skip the first sentence.

**Review by berthoug — CHANGES_REQUESTED** [review]

File: docs/hopsworksai/api_key.md (hunk 4 above)
Comment (maismail) [reply docs/hopsworksai/api_key.md:48]: Yes, It is not published yet

File: docs/hopsworksai/api_key.md (hunk 3 above)
Comment (maismail) [reply docs/hopsworksai/api_key.md:29]: done

**Review by berthoug — APPROVED** [review]

**Pull request merged**